src/main/java/com/questhelper/quests/sleepinggiants/SleepingGiants.java

```diff
@@ -0,0 +1,300 @@
+		polishingStoneFixed = new VarbitRequirement(13906, 2, Operation.EQUAL);
+		hammerFixed = new VarbitRequirement(13904, 2, Operation.EQUAL);
+
+		commissionReceived = new VarbitRequirement(13903, 10, Operation.EQUAL);
+		crateSearched = new VarbitRequirement(13903, 25, Operation.LESS_EQUAL);
+		shouldSetMould = new VarbitRequirement(13903, 30, Operation.LESS_EQUAL);
+		mouldSet = new VarbitRequirement(13903, 35, Operation.EQUAL);
+		shouldPourMetal = new VarbitRequirement(13903, 40, Operation.EQUAL);
+		metalPoured = new VarbitRequirement(13903, 45, Operation.EQUAL);
+
+		selectingMould = new WidgetPresenceRequirement(718, 2);
+		noForteSelected = new WidgetSpriteRequirement(718, 21, 3593);
+		noBladeSelected = new WidgetSpriteRequirement(718, 22, 3593);
+		noTipSelected = new WidgetSpriteRequirement(718, 23, 3593);
+		forteTabOpen = new WidgetSpriteRequirement(718, 12, 1, 297);
+		bladeTabOpen = new WidgetSpriteRequirement(718, 12, 10, 297);
+		tipTabOpen = new WidgetSpriteRequirement(718, 12, 19, 297);
+	}
+
+	public void setupSteps()
+	{
+		goToDesertPlateau = new DetailedQuestStep(this, new WorldPoint(3439, 9225, 0), "Go to the Desert Plateau to start the quest.", oakLogs, wool, nails, chisel, iceGloves);
+		((DetailedQuestStep) goToDesertPlateau).setLinePoints(Arrays.asList(new WorldPoint(3293, 3180, 0), new WorldPoint(3306, 3180, 0), new WorldPoint(3313, 3171, 0), new WorldPoint(3320, 3171, 0), new WorldPoint(3333, 3175, 0), new WorldPoint(3339, 3175, 0), new WorldPoint(3343, 3173, 0), new WorldPoint(3353, 3173, 0), new WorldPoint(3364, 3184, 0), new WorldPoint(3376, 3184, 0), new WorldPoint(3381, 3178, 0), new WorldPoint(3381, 3175, 0), new WorldPoint(3375, 3170, 0)));
+
+		takeHammer = new DetailedQuestStep(this, new WorldPoint(3350, 3162, 0), "Take a hammer from the crate north of the Giants' Foundry.");
+		takeHammer.addIcon(ItemID.HAMMER);
+
+		strikeHillGiant = new NpcStep(this, 11467 /* Hill Giant */, new WorldPoint(3361, 3147, 0), "Attempt to Strike the Hill Giant to start the quest.");
+		speakToKovac = new NpcStep(this, 11468 /* Kovac */, new WorldPoint(3361, 3147, 0), "Speak to Kovac.");
+
+		enterFoundry = new ObjectStep(this, 44635 /* Cave */, new WorldPoint(3361, 3150, 0), "Enter the Giants' Foundry.");
+
+		fixPolishingStone = new ObjectStep(this, 44775 /* Broken polishing wheel */, new WorldPoint(3363, 11485, 0), "Fix the Broken polishing wheel", twoOakLogs, fiveNails, hammer);
+		fixPolishingStone.addDialogStep("Yes.");
+		fixGrindstone = new ObjectStep(this, 44774 /* Broken grindstone */, new WorldPoint(3362, 11492, 0), "Fix the Broken grindstone", chisel);
+		fixGrindstone.addDialogStep("Yes.");
+		fixHammer = new ObjectStep(this, 44773 /* Broken trip hammer */, new WorldPoint(3364, 11497, 0), "Fix the Broken trip hammer", oneOakLog, fiveNails, hammer);
+		fixHammer.addDialogStep("Yes.");
+
+		speakToKovacAfterRepairs = new NpcStep(this, 11468 /* Kovac */, "Speak to Kovac to continue.");
```

Comment:
This NPC ID is incorrect. 
```suggestion
		speakToKovacAfterRepairs = new NpcStep(this, 11470 /* Kovac */, "Speak to Kovac to continue.");
```